packages/flutter_adaptive_scaffold/test/adaptive_layout_test.dart

```diff
@@ -415,7 +435,7 @@ AnimatedWidget leftInOut(Widget child, Animation<double> animation) {
   );
 }
 
-MediaQuery slot(double width, WidgetTester tester) {
+MediaQuery slot(double width, int durationMs, WidgetTester tester) {
```

Comment:
Why is this an int with the type in the name rather than a `Duration` that would directly encode unit information in the code?